platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -218,6 +219,6 @@ We love getting contributions from the Gradle community. For information on cont
 ## Reporting problems
 
 If you find a problem with this release, please file a bug on [GitHub Issues](https://github.com/gradle/gradle/issues) adhering to our issue guidelines.
-If you're not sure you're encountering a bug, please use the [forum](https://discuss.gradle.org/c/help-discuss).
+If you're not sure if you're encountering a bug, please use the [forum](https://discuss.gradle.org/c/help-discuss).
```

Comment:
💅 probably should be fixed in the template as well

Reply:
Done.